packages/rsocket-adapter-async/src/lib/requesters/fireAndForget.ts

```diff
@@ -0,0 +1,34 @@
+import { Codec } from "@rsocket/messaging";
+import { RSocket } from "@rsocket/core";
+import {
+  encodeCompositeMetadata,
+  WellKnownMimeType,
+} from "@rsocket/composite-metadata";
+
+export default function fireAndForget<TData>(
+  data: TData,
+  inputCodec: Codec<TData>
+): (
+  rsocket: RSocket,
+  metadata: Map<string | number | WellKnownMimeType, Buffer>
+) => Promise<void> {
+  return (
+    rsocket: RSocket,
+    metadata: Map<string | number | WellKnownMimeType, Buffer>
+  ) => {
+    const payload = {
+      data: data ? inputCodec.encode(data) : Buffer.allocUnsafe(0),
```

Comment:
I think we are going to have an issue with browser compatibility referencing `Buffer` globally, as it won't be available. We likely need to support `Buffer` to be injectable, but I think we have the same challenge in other packages at the moment as well.

Reply:
^-- we've concluded that this is a non concern and we expect consumers to need to polyfill Buffer.